### DISCUSSION TITLE
AIO installation not proceeding past login: No such container

### DISCUSSION THREAD

**Discussion by dandln** [nextcloud/all-in-one#1273] [Questions] [answered]
Hi,

I'm trying to install the AIO image (which is supposed to be running via an Apache Reverse Proxy in the future, but this is step 2) via Portainer, and I'm able to deploy the stack and start up the container just fine, and am able to access the setup page via `<IP>:8080`. It tells me my password and prompts me to log in, after which I seem to be redirected to `<IP>:8080/containers`, where I'm being hit by this error message:

```
Slim Application Error
The application could not run because of the following error:

Details
Type: GuzzleHttp\Exception\ClientException
Code: 404
Message: Client error: `POST http://localhost/v1.41/networks/nextcloud-aio/connect` resulted in a `404 Not Found` response: {"message":"No such container: nextcloud-aio-mastercontainer"}
File: /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php
Line: 113
```

Now the error message is quite clear in that it tries to access the master container but can't find it, however I don't understand what I did wrong, as I followed everything as best as I can, and I can't seem to find any similar issues here either.

Anyone know what might be going on here?

**Comment by szaimen** [comment, marked as answer]
It seems like you renamed the mastercontainer which is not allowed. It must be named `nextcloud-aio-mastercontainer`.

**Reply by dandln** [reply]
Almost, but not quite! Turned out that for some reason I can't explain my docker was configured in Swarm mode which made Portainer deploy the stack in Swarm mode, which in turn spawned the master container not with it's actual name but with weird, always changing names. Leaving my accidental swarm and deploying the stack again fixed it.
